ui/app/templates/components/oidc/scope-form.hbs

```diff
@@ -39,7 +39,7 @@
       You can use Alt+Tab (Option+Tab on MacOS) in the code editor to skip to the next field. See
       <button
         type="button"
-        class="text-button"
+        class="text-button has-text-info"
```

Comment:
This preserves the styling after removing `color: $link` from the `text-button` class.